docs/fairness.rst

```diff
@@ -0,0 +1,94 @@
+.. _vision_challenges:
+
+FAIRness of research workflows
+==============================
+
+.. image:: ./../img/fairness.png
+  :width: 500
+  :alt: TODO: caption
+
+As mentioned in the :ref:`introduction`, an overarching goal is to make research
+software `FAIR <https://www.go-fair.org/fair-principles/>`_. In this section, we
+therefore want to discuss possible aspects of FAIRness associated with the code
+and data that are used in or produced by scientific inquiry. A scientific workflow
+may consist of a variety of different pieces of software as well as input data,
+which makes it nontrivial to publish it in a comprehensive, reusable and reproducible
+way.
+
+.. contents::
+
+.. _transparency:
+
+Increasing transparency
+------------------------
+
+By publishing the data and software used in scientific investigations, researchers
+give their peers the opportunity to retrace the steps that were necessary to arrive
+at the conclusions drawn in a publication. This significantly increases the
+transparency of their research and besides this, it allows others to reuse the data
+and software in proceeding investigations.
+
+
+.. _reusable data:
+
+Reusable data
+-------------
+
+Integration of externally published research data in a research project may be
+cumbersome, if the data is not well-documented or self-explanatory. To mitigate
+this, data should be published in standard formats that are widely used in the
+community to guarantee their seamless integration into other projects. Moreover,
+rich metadata should be provided, which explains the contents of the data in
+detail and in a machine-readable format.
```

Comment:
@dglaeser the second time reading this I am a bit confused. What you describe is that essentially the data used should be FAIR, right? Should we rather call it 'FAIR data' instead of 'Reusable data'? the data should be FAIR in order for the research workflow to be FAIR ...